rootfs/deploy.py

```diff
@@ -36,6 +36,39 @@ def log(msg):
         print(msg)
 
 
+def get_registry_name():
+    hostname = os.getenv('DEIS_REGISTRY_HOSTNAME', "").replace("https://", "")
+    if regsitryLocation == "off-cluster":
+        organization = os.getenv('DEIS_REGISTRY_ORGANIZATION')
+        regName = ""
```

Comment:
can you put a comment here describing what the intention of this code is trying to do?